### DISCUSSION TITLE
Tooltip hide not working

### DISCUSSION THREAD

**Discussion by Mohammad99k** [twbs/bootstrap#36599] [Q&A]
Hi
I want to hide tooltip after user clicked on it.
```
el.addEventListener('click', function (e) {
            tooltip.hide()
})
```
Above not work in touch devices and tooltip still remain after click button.
How I can hide it?

**Comment by AnupamKhosla** [comment]
@Mohammad99k Did you change the default `trigger` option from `hover focus` to `click`? Can you add a working jsfiddle to replicate the issue?
